test/e2e/gatekeeper_controller_test.go

```diff
@@ -824,8 +832,8 @@ var _ = Describe("Gatekeeper", func() {
 					v, ok := annotations["service.beta.openshift.io/serving-cert-secret-name"]
 					g.Expect(ok).Should(BeTrue(),
 						"Should have service.beta.openshift.io/serving-cert-secret-name annotation")
-					g.Expect(v).Should(Equal("gatekeeper-webhook-server-cert"),
-						"Should be gatekeeper-webhook-server-cert")
+					g.Expect(v).Should(Equal(controllers.OpenshiftSecretName),
+						"Should be "+controllers.OpenshiftSecretName)
```

Comment:
The failure will already show the actual and expected strings, so I think a more helpful description would be what the test is about, something like:
```suggestion
					g.Expect(v).Should(Equal(controllers.OpenshiftSecretName),
						"service.beta.openshift.io/serving-cert-secret-name should have the expected value")
```